pages/home/build-tools/create-a-toolkit.mdx

```diff
@@ -40,37 +35,46 @@ Before starting, make sure you have:
 Run the following command to start creating your toolkit:
 
 ```bash
-git init
-arcade new
+uv tool run --from arcade-ai arcade new arithmetic
```

Comment:
`uv tool run --from arcade-ai arcade new arithmetic` executes the arcade CLI from the arcade-ai package in a disposable uv-managed virtual environment without first installing the package system-wide